src/itemlistformaction.cpp

```diff
@@ -156,7 +157,22 @@ bool ItemListFormAction::process_operation(Operation op,
 		LOG(Level::INFO, "ItemListFormAction: opening item at pos `%u'", itempos);
 		if (!visible_items.empty() && itempos < visible_items.size()) {
 			auto item = visible_items[itempos].first;
-			if (!open_item_in_browser(item)) {
+			const bool interactive = true;
+			if (!open_item_in_browser(item, interactive)) {
+				return false;
+			}
+			invalidate(itempos);
+		} else {
+			v->show_error(_("No item selected!"));
+		}
+	}
+	break;
+	case OP_OPENINBROWSER_NONINTERACTIVE: {
+		LOG(Level::INFO, "ItemListFormAction: opening item at pos `%u'", itempos);
```

Comment:
I think it'd be nice if the logging statements mentioned if they refer to interactive or non-interactive operation. It's just a log, so I think it'd be fine to do something like `"interactive: %u", static_cast<int>(interactive)`.

Reply:
Done:
- https://github.com/newsboat/newsboat/pull/1335/commits/ed6991651d579bd8d5fd9fcd397e82f3ec3f292f#diff-d5a1b4371967aac8adde72ce6f97d087e4f8d305f7176749ecdaf22d2ad62844R761
- https://github.com/newsboat/newsboat/pull/1335/commits/80f1766b060b0b2d6d7cb64737cd4ca01f4d2270#diff-1551378fbd86d2c7065b34517b3d6746dc86e657464942650be7078863458be3R547